config/muxcable.py

```diff
@@ -13,11 +14,11 @@
 
 REDIS_TIMEOUT_MSECS = 0
 
-CONFIG_SUCCESSFUL = 100
+CONFIG_SUCCESSFUL = 0
 CONFIG_FAIL = 1
 
 VENDOR_NAME = "Credo"
-VENDOR_MODEL = "CAC125321P2PA0MS"
+VENDOR_MODEL_REGEX = re.compile("CAC\w{3}321P2P\w{2}MS")
```

Comment:
Suggest making this a raw string: `re.compile(r"CAC\w{3}321P2P\w{2}MS")`

Reply:
fixed